src/test/skript/junit/EffCancelEvent.sk

```diff
@@ -0,0 +1,21 @@
+test "SimpleJUnitTest" when running JUnit:
```

Comment:
could you please also add a test for the swim event error (maybe the player login one too if you want)? you can use the parse structure to ensure it errors as expected